portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +25,25 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+	ArrayList<String> cities = new ArrayList<>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Welcome to my page");
+		String json = convertToJson();
```

Comment:
The formatting is looking really strange in Github because tabs are being combined with spaces. Would you please use spaces uniformly?

Reply:
gotchu!

Reply:
I was wondering why, I didn't see any review yesterday, turns out I forgot to create a new pull request lol